data-tool/flows/corps_tombstone_flow.py

```diff
@@ -437,12 +440,20 @@ def tombstone_flow():
 
             print(f'👷 Start processing {len(corp_nums)} corps: {", ".join(corp_nums[:5])}...')
 
-            users_mapper = migrate_corp_users(colin_engine, lear_engine, corp_nums)
-
-            # TODO: skip the following migration or continue?
-            if users_mapper is None:
-                print(f'❗ Skip populating user info for corps in this round due to user migration error.')
-                users_mapper = {}
+            try:
+                users_mapper = migrate_corp_users(colin_engine, lear_engine, corp_nums)
+            except Exception as e:
+                # skip migration if there's user migration error
```

Comment:
Not sure if we should fail and stop the corp migration in this round if the user migration fails.

Reply:
yeah, i think we should fail and stop the corp migration if user migration fails